client/my-sites/themes/fse-themes.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { useQuery } from 'react-query';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'calypso/components/empty-content';
+import wpcom from 'calypso/lib/wp';
+import { ConnectedThemesSelection } from './themes-selection';
+
+const FseThemes = localize( ( { translate, ...restProps } ) => {
+	const { data, error, isLoading } = useQuery(
+		'fse-themes',
+		() =>
+			wpcom.req.get( '/themes', {
+				filter: 'block-templates',
+				number: 50,
```

Comment:
We should probably figure out how to standardize this number so we don't have to update it in multiple places. 🙃 Not a blocker, though.